### PR TITLE
[Homepage] bug: fixed too much whitespace in banner4

### LINKED ISSUE #3686: [Homepage] Banner4: Not properly responsive; leaving too much whitespace.
### Description
On the site homepage, on banner4, please see the screenshot for a description of the issue.

### Expected Behavior
These two sections should be closer together at all times.

### Screenshots
Screenshot 2023-02-02 at 7 08 36 PM

### Environment:
- Host OS: 
- Browser: 

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Oghene-kevwe**
Signed-off-by: Idise Praise <Idisepraise@gmail.com>

**Description**

This PR fixes #3686

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@ShuvamChatterjee28 would you like offer review here?

**Comment by l5io** [comment]
🚀 Preview for commit a6df39b5b3a28f258df9416314a0851c178b4c1a at: https://63e1822aa0965b1c1fa0bd45--layer5.netlify.app

**Comment by ShuvamChatterjee28** [comment]
Hello @leecalcote, certainly I would love to review this one. Thanks.

**Comment by ShuvamChatterjee28** [comment]
LGTM, I am approving the changes.

**Review by ShuvamChatterjee28 — APPROVED** [review]

**Review by Shivam-AfA — APPROVED** [review]
Looks good.

**Review by leecalcote — APPROVED** [review]
Thanks, @Oghene-kevwe, all. 👍

**Review by leecalcote — CHANGES_REQUESTED** [review]
I still see the following behavior in Safari:

Screenshot 2023-02-08 at 4 25 18 AM
Screenshot 2023-02-08 at 4 25 28 AM

**Comment by leecalcote** [comment]
@Shivam-AfA keep a watchful eye, please.

@Oghene-kevwe, will you be fixing the outstanding issues screenshot'ed above?

**Comment by ShuvamChatterjee28** [comment]
@leecalcote I also see from the Safari, the section below the navbar is hooked. However in the chrome it is showing differently. I don't have much frontend knowledge, but I think it is causing because it is hooked with the navbar. Please correct if I am wrong.

**Comment by Shivam-AfA** [comment]
@leecalcote This is strange. It appears different on my pc. I guess I'll have to use cross check with Safari for UI based issues.

**Comment by Oghene-kevwe** [comment]
@leecalcote yes, I will definitely fix it

File: src/components/BGImage/BGImage.styles.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const Content = styled.div`
   height: 100%;
   width: 100%;
   z-index: 2;
-  @media (max-width: 768px){
-    top: 40%;
+  @media (max-width: 992px){
+    top: 35%;
```
Comment (Nikhil-Ladha) [inline src/components/BGImage/BGImage.styles.js:32]: The fix is not simply updating the `top` value. It requires drilling through the CSS of the `BGImage` component and updating the `position` value to `relative` and `absolute` based on resolution.

**Comment by leecalcote** [comment]
This is a relatively urgent issue.

**Comment by Nikhil-Ladha** [comment]
I will wait till afternoon tomorrow, if @Oghene-kevwe doesn't push a fix by then will make the update myself.

**Comment by ShuvamChatterjee28** [comment]
Hey, I would like to work on this one. If I can't complete it within tomorrow mid-night, then please feel free to complete it. This will be a good chance for me to learn CSS.

**Comment by Oghene-kevwe** [comment]
@leecalcote @Nikhil-Ladha  I'm sorry for the late response. I will push a fix before tomorrow.

**Comment by ShuvamChatterjee28** [comment]
Hey, I can see a good part of my fix is here. Thank you @Nikhil-Ladha for adding the suggested part. I have learned a lot to solve this one. Very happy to see it. :)

**Comment by Nikhil-Ladha** [comment]
> Hey, I can see a good part of my fix is here. Thank you @Nikhil-Ladha for adding the suggested part. I have learned a lot to solve this one. Very happy to see it. :)

Yeah, np :)

**Comment by l5io** [comment]
🚀 Preview for commit b56b07bcfbafaad8febcad70b32c3ff8ebc31cda at: https://63e9db57ccbb6b2815b19f83--layer5.netlify.app

**Comment by ShuvamChatterjee28** [comment]
@Nikhil-Ladha #3745 and #3691 is also fixed with this fix.

**Pull request merged**